### DISCUSSION TITLE
Directory merging

### DISCUSSION THREAD

**Discussion by UKenGB** [trapexit/mergerfs#1181] [Q&A]
Is it possible to use mergerfs to merge multiple directories on to a single mountpoint, rather than multiple filesystems?

I'm using mergerfs to merge some filesystems and it's great, but I also have several directories for which I want to do the same thing, i.e. have a single directory that apparently contains everything from those multiple directories.

Is this possible with mergerfs?

**Comment by trapexit** [comment]
Yes. There is no difference between the two. A filesystem is just mounted to a directory. 

I specifically handle any situations where being on the same filesystem may matter as mentioned https://github.com/trapexit/mergerfs#statfs--statvfs
